macOS/DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -221,6 +226,10 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
             fileStore = EncryptedFileStore()
         }
 
+        appearancePreferences = AppearancePreferences(keyValueStore: keyValueStore)
```

Comment:
@ayoy  This looks good, there is one interesting addition we should think about: it is possible the first read will fail, so we could (at the place we setup kvs) add dummy read, and crash up on failure - this way consumers will always be able to read data successfully. But I'm leaving that up to you as I'm not sure what is desired startup failure handling on macOS.

Reply:
Thanks @bwaresiak! I’ve added a dummy check at the KVS initialization that will force a crash if throwing (that’s how macOS handles failures in persistent layer setup).